data/transaction.go

```diff
@@ -24,25 +24,32 @@ type Transaction struct {
-	Sender           string                 `json:"sender,omitempty"`
-	GasPrice         uint64                 `json:"gasPrice,omitempty"`
-	GasLimit         uint64                 `json:"gasLimit,omitempty"`
-	Data             []byte                 `json:"data,omitempty"`
-	Code             string                 `json:"code,omitempty"`
-	Signature        string                 `json:"signature,omitempty"`
-	SourceShard      uint32                 `json:"sourceShard,omitempty"`
-	DestinationShard uint32                 `json:"destinationShard,omitempty"`
-	BlockNonce       uint64                 `json:"blockNonce,omitempty"`
-	MiniBlockHash    string                 `json:"miniblockHash,omitempty"`
-	BlockHash        string                 `json:"blockHash,omitempty"`
-	Status           core.TransactionStatus `json:"status,omitempty"`
+	Type                              string                 `json:"type"`
+	Hash                              string                 `json:"hash,omitempty"`
+	Nonce                             uint64                 `json:"nonce,omitempty"`
+	Round                             uint64                 `json:"round,omitempty"`
+	Epoch                             uint32                 `json:"epoch,omitempty"`
+	Value                             string                 `json:"value,omitempty"`
+	Receiver                          string                 `json:"receiver,omitempty"`
+	Sender                            string                 `json:"sender,omitempty"`
+	GasPrice                          uint64                 `json:"gasPrice,omitempty"`
+	GasLimit                          uint64                 `json:"gasLimit,omitempty"`
+	Data                              []byte                 `json:"data,omitempty"`
+	Code                              string                 `json:"code,omitempty"`
+	Signature                         string                 `json:"signature,omitempty"`
+	SourceShard                       uint32                 `json:"sourceShard,omitempty"`
+	DestinationShard                  uint32                 `json:"destinationShard,omitempty"`
+	BlockNonce                        uint64                 `json:"blockNonce,omitempty"`
+	BlockHash                         string                 `json:"blockHash,omitempty"`
+	NotarizedAtSourceInMetaNonce      uint64                 `json:"notarizedAtSourceInMetaNonce,omitempty"`
+	NotarizedAtSourceInMetaHash       string                 `json:"NotarizedAtSourceInMetaHash,omitempty"`
+	NotarizedAtDestinationInMetaNonce uint64                 `json:"notarizedAtDestinationInMetaNonce,omitempty"`
+	NotarizedAtDestinationInMetaHash  string                 `json:"notarizedAtDestinationInMetaHash,omitempty"`
+	MiniBlockHash                     string                 `json:"miniblockHash,omitempty"`
+	Status                            core.TransactionStatus `json:"status,omitempty"`
+
+	HyperblockNonce uint64 `json:"hyperblockNonce,omitempty"`
```

Comment:
indent as others - is it not strange to get this transaction with some empty fields when asking normal - no hyperblock observers ?

Reply:
Hyperblock is composed at Proxy side, and the observers do not know the concept of a hyperblock.

But yes, for observers that do not have this extra indexing enabled (aka. `fullHistory.Enabled = true`), the fields will not be present in the response (`omitempty`).

Fixed indentation.